config/reasoner/dl2op/main/mapping.n3

```diff
@@ -121,6 +107,46 @@ PREFIX lblodlg: <http://data.lblod.info/vocabularies/leidinggevenden/>
   ?s mandaat:bindingEinde ?bindingEindeDate
 }.
 
+{
+  ?s ch:startdatum ?startDate.
+  # use backward arrow rule defined below
+  ?startDate :dateFromDateTime ?startDateTime.
+  {
+    ?s ch:startdatum ?startDate
+  } e:becomes {
+    ?s ch:startdatum ?startDateTime
+  }.
+} => {
+  ?s ch:startdatum ?startDateTime
+}.
+
+{
+  ?s ch:eindedatum ?startDDate.
+  # use backward arrow rule defined below
+  ?startDate :dateFromDateTime ?startDateTime.
+  {
+    ?s ch:eindedatum ?startDate
+  } e:becomes {
+    ?s ch:eindedatum ?startDateTime
+  }.
+} => {
+  ?s ch:eindedatum ?startDateTime
+}.
+
+{
+  ?s ere:geplandeEinddatumAanstelling ?startDDate.
```

Comment:
The `ere` prefix is not defined in this file, causing the consumers to crash:
```
leidinggevenden-consumer-1  | Statements:  ["{\"errors\":[{\"detail\":\"** ERROR ** file:///config/dl2op/main/mapping.n3 ** no_prefix_directive(ere,after_line(134))\\n\",\"status\":500}]}",""]
```

There is also a **D** too much in the object name.